### PR TITLE
Aj/bug/datapicker

### PR DESCRIPTION
**Pull request by JanAsaulenko**

### CODE REVIEW

File: app/Http/Controllers/MainController.php [hunk 1]
```diff
@@ -38,16 +38,25 @@ public function place(){
...
-    public function getPlace(Request $request)
+    public function getPlaces(Request $request)
     {
         $id_req = $request->city_id;
         $places = Place::all()->where('id_city', $id_req);
         return array_map(function($place){
             return array ('id' => $place->id, 'text' => $place->address);
         },$places->all());
     }
+    public function getPlaces2(Request $request)
```
Comment (AlexOrd) [inline app/Http/Controllers/MainController.php:49]: What do you mean at 'getPlaces2' ?

File: app/Http/Controllers/MainController.php [hunk 2]
```diff
@@ -38,16 +38,25 @@ public function place(){
...
+        $id_req = $request->place_id;
+
+        $places = Place::all()->where('id_city', $id_req);
+        return array_map(function($place){
+            return array ('id' => $place->id, 'text' => $place->address);
+        },$places->all());
+    }
 
-    public function getPlaceLocation(Request $request){
+    public function getSpaces(Request $request){
```
Comment (AlexOrd) [inline app/Http/Controllers/MainController.php:59]:
```
$name_places = NamePlace::all()->where('place_id', $id_req);
```
Can we have few plases with the same ID ?

Comment (JanAsaulenko) [reply app/Http/Controllers/MainController.php:59]: Yes, they all have the same id of the city. There is array of places.

File: app/Http/Controllers/MainController.php (hunk 1 above)
Comment (JanAsaulenko) [reply app/Http/Controllers/MainController.php:49]: It is templayte. because backend is not ready yet.

Comment (AlexOrd) [reply app/Http/Controllers/MainController.php:49]: Ok, then add some comments in the code.

File: app/Http/Controllers/MainController.php (hunk 2 above)
Comment (AlexOrd) [reply app/Http/Controllers/MainController.php:59]: We should rename this field in future (
Plz, add the comment in this part of code too.

**Review by AlexOrd — CHANGES_REQUESTED** [review]

**Pull request merged**